packages/phone-number-privacy/combiner/src/common/identity.ts

```diff
@@ -25,11 +25,15 @@ export function authenticateUser(request: Request): boolean {
 export async function isVerified(account: string, hashedPhoneNumber: string): Promise<boolean> {
   // TODO (amyslawson) wrap forno request in retry
   const attestationsWrapper: AttestationsWrapper = await getContractKit().contracts.getAttestations()
-  const attestationStatus = await attestationsWrapper.getVerifiedStatus(hashedPhoneNumber, account)
+  const {
+    isVerified: _isVerified,
+    completed,
+    numAttestationsRemaining,
+    total,
+  } = await attestationsWrapper.getVerifiedStatus(hashedPhoneNumber, account)
 
   logger.debug(
```

Comment:
Let's also log the `account` so that we can better correlate with a mobile session.